<ui version="4.0" >
 <class>QgsGraduatedSymbolDialogBase</class>
 <widget class="QDialog" name="QgsGraduatedSymbolDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>graduated Symbol</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="2" row="1" column="0" >
    <widget class="QStackedWidget" name="mSymbolWidgetStack" >
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="classvarlabel" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Classification Field:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="modelabel" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Mode:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="numberofclasseslabel" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>Number of Classes:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="classificationComboBox" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modeComboBox" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numberofclassesspinbox" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="mDeleteClassButton" >
     <property name="text" >
      <string>Delete class</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="mClassifyButton" >
     <property name="text" >
      <string>Classify</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QListWidget" name="mClassListWidget" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
